mne/io/brainvision/tests/test_brainvision.py

```diff
@@ -62,6 +63,33 @@
 event_id = {'Sync On': 5}
 
 
+def test_orig_units():
+    """Test exposure of original channel units."""
+    with pytest.warns(RuntimeWarning, match='will be dropped'):
+        raw = read_raw_brainvision(vhdr_path)
+    orig_units = raw._orig_units
+    assert len(orig_units) == 32
+    assert orig_units['FP1'] == u'µV'
+    assert orig_units['CP5'] == 'BS'
+    assert orig_units['CP6'] == u'µS'
+    assert orig_units['HL'] == 'ARU'
+    assert orig_units['HR'] == 'uS'
+    assert orig_units['Vb'] == 'S'
+    assert orig_units['ReRef'] == 'C'
+
+    # Also test handling in base.py
+    # Should complain that channel Cz does not have a corresponding original
+    # unit.
+    with pytest.raises(ValueError, match='has no associated original unit.'):
```

Comment:
also, would you know a better place for this test and the one on line 88 @larsoner?

both of them are general and not BrainVision specific ... they need to be run only once (so probably they should not be in `test_raw.py::_test_raw_reader`), but they could just as well be in EDF - or whatever file format that we expose the orig_units for in the future

Reply:
You can create a new `io/tests/test_base.py` if you want